src/public/client.ts

```diff
@@ -63,20 +62,29 @@ class DisplayMap {
       const editButton = helper.getElement('.edit', mappingElement)
       editButton.onclick = (): void => {
         mappingElement.innerHTML = `
-        <li class='list-group-item' style="display: flex;">
-         Domain: <input class=' form-control domain'type="" value='${data.domain}'>
-         Port:  <input class='form-control port'type="" value=${data.port}>
-         IP:  <input class='form-control ip' type="" value=${data.ip}>
-         <hr/>
-         <div class ='saveButtonContainer'>
-         <button  class='btn save' style="padding: 0px 0px 0px 15px">SAVE</button>
-         </div>
-         </li>
+        <li class='list-group-item'>
+        <form>
```

Comment:
I wonder if it would not be better to use onsubmit instead of onclick with a `<button type="submit">` ?